liri.js

```diff
@@ -0,0 +1,155 @@
+//Dependencies
+var keys = require("./keys.js");
+var fs = require("fs");
+var twitter = require('twitter');
+var Spotify = require('node-spotify-api');
+var request = require('request');
+
+
+//Select from 4 command options
+var command = process.argv[2];
+
+//Allows more than one word of text for command inputs
+var query = process.argv.slice(3).join(" ");
+
+
+//Determine which function to run
+main();
+function main() {
+  switch (command) {
+    case "my-tweets":
+      tweets();
+      break;
+
+    case "spotify-this-song":
+      spotify();
+      break;
+
+    case "movie-this":
+      imdb();
+      break;
+
+    case "do-what-it-says":
+      simon();
+      break;
+  }
+}
+
+//my-tweets
+function tweets() {
+  let client = new twitter({
+    consumer_key: keys.twitterKeys.consumer_key,
+    consumer_secret: keys.twitterKeys.consumer_secret,
+    access_token_key: keys.twitterKeys.access_token_key,
+    access_token_secret: keys.twitterKeys.access_token_secret
+  });
+  let params = {
+    screen_name: 'djlowes',
+    count: 20
+  };
+  client.get('statuses/user_timeline', params, function(error, tweets, response) {
+    if (!error && response.statusCode === 200) {
```

Comment:
It is better practice to test for errors so that you can handle them appropriately instead of handling not errors.